builder/packages.sls

```diff
@@ -26,3 +26,34 @@ builder-packages:
           - libllvm4.0
           - libllvm4.0:i386
           - llvm-4.0-dev
+          - curl
+            # Android lists this as a dependency
+          - distcc
+          - dosfstools
+          - gettext
+          - git-core
+            # Android lists this as a dependency
+          - gnupg
+          - gperf
+          - lib32ncurses5-dev
+          - lib32z-dev
+          - libc6-dev-i386
+          - libgl1--dev
+          - libncurses5-dev:i386
+          - libreadline6-dev:i386
+          - libxml2-utils
+          - lunch
+          - mtools
+          - openjdk-8-jdk
+          - python2.7
+          - python-mako
```

Comment:
python-mako is in salve, and python2.7 will be pulled in implicitly. I'd rather add this to slave if we're going to add it